assets/js/util/sanitize.js

```diff
@@ -8,3 +8,25 @@ export const sanitizeHTML = ( unsafeHTML, domPurifyConfig = {} ) => {
 		__html: purify.sanitize( unsafeHTML, domPurifyConfig ),
 	};
 };
+
+/**
+ * Takes a URL string, removes the trailing stash if any and returns it.
+ *
+ * @since 1.18.0
+ * @private
+ *
+ * @param {string} url The URL with or without trailing slash.
+ * @return {string|null} The URL string after removing the trailing slash.
+ */
```

Comment:
A few points here:
* This utility shouldn't be URL-specific. Like it's PHP counterpart, it's a basic string utility.
* The `@since` annotation should use the `n.e.x.t` placeholder
* I feel like the `T` maybe shouldn't be capitalized, but not a big deal